tests/unit/test_runners.py

```
@@ -596,3 +599,150 @@ def test_self_errors_includes_chunk_errors(self):
+		from secator.hooks.gcs import upload_resume_file
+
+		with tempfile.NamedTemporaryFile(suffix='.cfg', delete=False) as f:
+			f.write(b'resume_data')
+			resume_path = f.name
+
+		cp = Checkpoint(task_id='t1', task_name='httpx_1', resume_file_path=resume_path)
+		mock_self = MagicMock()
+		mock_self.threads = []
+
+		with patch('secator.hooks.gcs.GCS_BUCKET_NAME', 'test-bucket'), \
+				patch('secator.hooks.gcs.upload_blob') as mock_upload:
```

Flake8 (7.3.0)

[error] continuation line over-indented for visual indent

(E127)